jetbrains-core/src/software/aws/toolkits/jetbrains/core/tools/ToolConfigurable.kt

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.tools
+
+import com.intellij.openapi.fileChooser.FileChooserDescriptorFactory
+import com.intellij.openapi.options.BoundConfigurable
+import com.intellij.openapi.options.SearchableConfigurable
+import com.intellij.ui.components.JBTextField
+import com.intellij.ui.layout.applyToComponent
+import com.intellij.ui.layout.panel
+import software.aws.toolkits.jetbrains.ui.ValidatingPanel
+import software.aws.toolkits.resources.message
+import java.nio.file.Path
+
+class ToolConfigurable : BoundConfigurable(message("executableCommon.configurable.title")), SearchableConfigurable {
+    private val settings = ToolSettings.getInstance()
+    private val manager = ToolManager.getInstance()
+    private val panel by lazy {
+        ValidatingPanel(
+            disposable!!,
```

Comment:
add comment on why this is safe?

Reply:
did you one better and made it into an exception